### ISSUE TITLE
panic: glGetActiveSubroutineName

### ISSUE THREAD

**Issue by vexe** [OpenBiohazard2/OpenBiohazard2#5]
Hi there,

I'm trying to get the executable to run (Win10 64-bit), I downloaded Go and got Msys2. I ran 'go build' and it built fine, when I run the executable however I get:

```
panic: glGetActiveSubroutineName

goroutine 1 [running, locked to thread]:
github.com/samuelyuan/openbiohazard2/render.InitRenderer(0x400, 0x300, 0x5bb7da)
        H:/OpenBiohazard2-master/render/rendermain.go:49 +0x5b1
main.main()
        H:/OpenBiohazard2-master/main.go:32 +0xe5
```

Is there a step I'm missing?

Thanks for you help and awesome work!

**Comment by samuelyuan** [comment]
The error message means gl.Init() failed. What version of OpenGL are you using? The code is using OpenGL 4.1.

**Comment by vexe** [comment]
Hey thanks for the reply! I was using the same OpenGL 4.1 as I didn't modify anything in the code when I built it. I tried higher versions (by changing all the GL4.1 github import links) but they had the same issue. The one that worked for me was 3.3 for some reason.

The other thing, it looks like this implementation currently only supports pal versions, my data was jap and I just had to modify the const paths in the resources file to point to my data, should probably just add a note for that.

Great work!
